README.md

```diff
@@ -377,6 +377,29 @@ file we have added a lot of comments into the code to make it easy to;
 
 Even with lots of comments we make to support more detailed documention.  You will find the documatation we have on the [Wiki pages] (https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/wiki).  Missing documentation you want?  Start a page and/or open an [issue] (https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/issues) to get it added.
 
+
+HOW TO VISUALIZE PerfKitBenchmarker RESULTS
```

Comment:
PerfKitExplorer isn't the *only* way to visualize the results. Maybe make this "Visualizing the results in PerfKitExplorer"?